doc/material/index.rst

```diff
@@ -190,36 +268,30 @@ The code relies on the following input files, stored in :file:`data/material/`:
 :file:`iea_mappings/industry.csv`
    Mapping of MESSAGEix-GLOBIOM industry technologies to IEA EWEB products of industry sector flows not covered by MESSAGEix-Materials
 
-:file:`ammonia/demand_NH3.yaml`
-   Ammonia demand in each R12 region in year 2020
-
 :file:`other/mer_to_ppp_default.csv`
    Default conversion factors for GDP MER to PPP used in MESSAGEix-GLOBIOM SSP2 scenarios.
    Used to create demand projections for steel/cement/aluminum/chemicals if GDP_PPP data is not in scenario
 
-:file:`buildings/LED_LED_report_IAMC_sensitivity_R12.csv`
-   Data from MESSAGEix-Buidings LED scenarios used to get steel/cement/aluminum demands from buildings
 
-:file:`buildings/report_IRP_SSP2_BL_comm_R12.csv`
-   Commerical sector data from MESSAGEix-Buidings used to get steel/cement/aluminum demands from buildings
 
-:file:`buildings/report_IRP_SSP2_BL_resid_R12.csv`
-   Residential sector data from MESSAGEix-Buidings used to get steel/cement/aluminum demands from buildings
+:file:`material/set.yaml`
+----------------------------
 
-:file:`/methanol/results_material_SHAPE_comm.csv`
-   Commercial sector data from MESSAGEix-Buidings SHAPE scenario used to get wood demands from buildings to estimate resin demands
+.. literalinclude:: ../../message_ix_models/data/material/set.yaml
+   :language: yaml
 
-:file:`/methanol/results_material_SHAPE_resid.csv`
-   Residential sector data from MESSAGEix-Buidings SHAPE scenario used to get wood demands from buildings to estimate resin demands
 
-:file:`/methanol/results_material_SSP2_comm.csv`
-   Commercial sector data from MESSAGEix-Buidings SSP2 scenario used to get wood demands from buildings to estimate resin demands
+Release notes
+-----------
 
-:file:`/methanol/results_material_SSP2_resid.csv`
-   Residential sector data from MESSAGEix-Buidings SSP2 scenario used to get wood demands from buildings to estimate resin demands
 
-:file:`material/set.yaml`
-----------------------------
+This is the list of changes to pandas between each release. For full details,
+see the `commit logs <https://github.com/pandas-dev/pandas/commits/>`_. For install and
+upgrade instructions, see :ref:`install`.
 
-.. literalinclude:: ../../message_ix_models/data/material/set.yaml
-   :language: yaml
+
+**Version 3.0**
```

Comment:
This seems like a leftover from the pandas inspiration; if so: please clean this up :)